src/orion/core/worker/experiment.py

```diff
@@ -1,4 +1,4 @@
-# pylint:disable=protected-access,too-many-public-methods,too-many-lines
+# pylint:disable=protected-access,too-many-public-methods,too-many-lines,too-many-branches
```

Comment:
Is this exception specific to `stats` property? If yes it should be added there specifically.

Reply:
Ok!